README.md

```diff
@@ -19,6 +19,8 @@
 
 #### Build Gaia (Cosmos SDK)
 
+Note: the following requires `docker` installed.
+
```

Comment:
Thank you, the problem is that the Docker instructions are farther down in the README.  It would be better to move them up.